### ISSUE TITLE
Display wallpaper next to other widgets

### ISSUE THREAD

**Issue by DocKDE** [kolbyjack/MMM-Wallpaper#90]
I realize this is probably not exactly the intended use case of the module but I would like to display the wallpaper as a regular image widget next to other widgets I use (calendar, weather etc.). I assume this should be possible but I'm not much of a CSS wizard to make it happen. Is there a setting for this?

Thanks for your work!

**Comment by kolbyjack** [comment]
Yes, you can set `fillRegion` to `false`, and then set a specific `width` and `height`:

```json
{
  "source": "whatever",
  "fillRegion": false,
  "width": "300px",
  "height": "300px"
}
```

**Comment by DocKDE** [comment]
Ah, I see. Seems to work well enough, thanks for the reply!
